CONTRIBUTING.md

```diff
@@ -23,12 +25,9 @@ Note that if an issue isn't a high priority or has many open questions then it m
 The ONNX Runtime team will review the proposal and either approve or close the issue based on whether it broadly aligns with the [Onnx Runtime Roadmap - High Level Goals section](./docs/Roadmap.md) and contribution guidelines.
 
 4. **API Review**  
-If the feature adds new APIs then we'll start an API review.
-All new public APIs must be reviewed before merging.  
-For making changes to the C API refer to guidance [here](onnxruntime/core/session/onnxruntime_c_api.cc#L1326).
-For making changes to the WinRT API someone from the ONNX Runtime team will work with you.
+If the feature adds new APIs then we'll start an API review. All new public APIs must be reviewed before merging. For making changes to the C API refer to guidance [here](onnxruntime/core/session/onnxruntime_c_api.cc#L1326). For making changes to the Windows Machine Learning WinRT API, please label your request as *api:WinML*, and someone from the Windows ML team will work with you. 
```

Comment:
>L1326 [](start = 217, length = 5)

since this line is changing, can you fix it to the right line number.   [https://github.com/alexzakv/onnxruntime/blob/user/alexzak/WinML_principles/onnxruntime/core/session/onnxruntime_c_api.cc#L1813](https://github.com/alexzakv/onnxruntime/blob/user/alexzak/WinML_principles/onnxruntime/core/session/onnxruntime_c_api.cc#L1813)

Reply:
numbers fixed